PIL/Image.py

```diff
@@ -588,11 +588,11 @@ def _dump(self, file=None, format=None):
 
         self.load()
         if not format or format == "PPM":
-            self.im.save_ppm(file)
+            self.im.save_ppm(file, **options)
```

Comment:
Sorry, I was wrong here, options aren't actually accepted for the save_ppm internal method.